packages/react-native-editor/android/app/src/main/java/com/gutenberg/MainActivity.java

```diff
@@ -27,6 +29,13 @@ protected ReactActivityDelegate createReactActivityDelegate() {
             @Override
             protected Bundle getLaunchOptions() {
                 Bundle bundle = new Bundle();
+
+                // Add locale
+                String languageString = Locale.getDefault().toString();
+                String localeSlug = languageString.replace("_", "-").toLowerCase(Locale.ENGLISH);
+                bundle.putString(GutenbergProps.PROP_LOCALE, localeSlug);
```

Comment:
Looks like the Gutenberg demo app wasn't providing the locale to the editor, following how we fetch it in the WordPress app ([reference](https://github.com/wordpress-mobile/WordPress-Utils-Android/blob/1ed207c03d2242b6fc3d74f9e388e9163cbc82a6/WordPressUtils/src/main/java/org/wordpress/android/util/LanguageUtils.java#L27)), I added the locale property so we can test translations.